src/style.css

```diff
@@ -0,0 +1,27 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background: url(./images/star.svg) no-repeat, #fff;
+  background-position: center;
+}
+
+.stars__star:not(:last-child) {
+  margin-right: 4px;
+}
+
+.stars--1 :first-child,
+.stars--2 :not(:nth-child(n+3)),
+.stars--3 :not(:nth-child(n+4)),
+.stars--4 :not(:nth-child(n+5)),
+.stars--5 * {
+  background: url(./images/star-active.svg) no-repeat, #fff;
+  background-position: center;
```

Comment:
remove background-position, it doesn't change
```suggestion
```

Reply:
Linter result error without background-position, what will I need do?